### PR TITLE
fixing bundling of cuit dlls in testplatform nuget

### PR DESCRIPTION
**Pull request by acesiddhu**
## Description
Bundling cuit dlls to testplatform nuget package

### CODE REVIEW

**Review by singhsarab — APPROVED** [review]

File: src/package/nuspec/Microsoft.TestPlatform.nuspec [hunk 1]
```diff
@@ -388,5 +390,38 @@
...
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.Interop.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Extension.IE.Communication.Interop.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Playback.Engine.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Playback.Engine.dll" />
+    <file src="net451\$Runtime$\Microsoft.VisualStudio.TestTools.UITest.Synchronization.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.TestTools.UITest.Synchronization.dll" />
+    <file src="net451\$Runtime$\UIAComWrapper.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\UIAComWrapper.dll" />
```
Comment (abhishkk) [inline src/package/nuspec/Microsoft.TestPlatform.nuspec]: spacing?

File: src/package/nuspec/Microsoft.TestPlatform.nuspec [hunk 2]
```diff
@@ -388,5 +390,38 @@
...
+
+    <file src="net451\$Runtime$\IEPropertyConfiguration.xml"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\CUITPlugins\IEPropertyConfiguration.xml" />
```
Comment (abhishkk) [inline src/package/nuspec/Microsoft.TestPlatform.nuspec]: Should it not be src="net451\$Runtime$\CUITPLugins\IEPropertyConfiguration.xml"

Comment (acesiddhu) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: good catch. it failed locally fixing it

File: src/package/nuspec/Microsoft.TestPlatform.nuspec (hunk 1 above)
Comment (acesiddhu) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: it has spaces on my vs code :(

**Review by abhishkk — APPROVED** [review]

**Pull request merged**